packages/x-charts-pro/src/FunnelChart/funnel.types.ts

```diff
@@ -124,6 +124,7 @@ export interface DefaultizedFunnelSeriesType
   extends DefaultizedProps<FunnelSeriesType, CommonDefaultizedProps | 'layout'> {
   dataPoints: FunnelDataPoints[][];
   data: Readonly<MakeRequired<FunnelValueType, 'id' | 'color'>[]>;
+  dataDirection: 'increasing' | 'decreasing';
```

Comment:
"Increasing" means that the value increases from top to bottom or from bottom to top? What about an horizontal funnel? We should probably specify that in a comment here to make it clear.

Reply:
Increasing means the value is smaller to bigger, regardless of orientation. I'll add a doc